include/openssl/x509v3.h

```diff
@@ -863,6 +863,53 @@ int X509v3_addr_validate_resource_set(STACK_OF(X509) *chain,
 
 #endif                         /* OPENSSL_NO_RFC3779 */
 
+DEFINE_STACK_OF(ASN1_STRING)
+
+typedef struct NamingAuthority_st NAMING_AUTHORITY;
+
+typedef struct ProfessionInfo_st PROFESSION_INFO;
+
+typedef struct Admissions_st ADMISSIONS;
+
+typedef struct AdmissionSyntax_st ADMISSION_SYNTAX;
+
+DECLARE_ASN1_FUNCTIONS(NAMING_AUTHORITY)
+DECLARE_ASN1_FUNCTIONS(PROFESSION_INFO)
+DECLARE_ASN1_FUNCTIONS(ADMISSIONS)
+DECLARE_ASN1_FUNCTIONS(ADMISSION_SYNTAX)
+
```

Comment:
Q: do we need to define all the ASN.1 i2d/d2i functions for this? They're handled internally by the extension code and I can't see any reason why an application would want/need to encode/decode them in any other context. If so these can be pruned so we only have the allocation/deallocation functions instead of everything.

Reply:
For me it`s fine to remove these code.